src/ui/components/WorkbookIcon/WorkbookIcon.scss

```diff
@@ -2,12 +2,12 @@
     display: flex;
     align-items: center;
     justify-content: center;
-    background-color: #ffe7e7;
-    border-radius: 6px;
+    background-color: var(--g-color-base-workbook-icon);
+    border-radius: 12px;
     text-transform: uppercase;
 
     &__letters {
         text-align: center;
-        color: #000;
+        color: currentColor;
```

Comment:
In what cases is this needed?

Reply:
unnecessary, work correct in different themes without color